glean-core/ios/Glean/Glean.swift

```diff
@@ -1,25 +1,67 @@
-//
-//  Glean.swift
-//  Glean
-//
-//  Created by Jan-Erik Rediger on 27.03.19.
-//  Copyright © 2019 Jan-Erik Rediger. All rights reserved.
-//
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
 
+/// The main Glean API.
+///
+/// This is exposed through the global [Glean.shared] object.
 public class Glean {
     public static let shared = Glean()
 
     private var initialized: Bool = false
+    private var handle: UInt64 = 0
+    private var uploadEnabled: Bool = true
```

Comment:
If we want to access this via tests, then we may need to relax this to the default access of `internal`. If we stick with `private` it doesn't allow us to access this in tests.  I'm okay leaving it as is for now but just wanted to give a heads up on this.

Reply:
Ah well, we should change that then indeed, at least for `handle` access is necessary